auth.py

```diff
@@ -2,6 +2,7 @@
 from .resources import Message
 from .token  import Token
 from .settings import Settings
+import exceptions as ResinException
```

Comment:
This will explode in Python 3. There's no immediate need for Python 3 support, as other parts of the code also break on Python 3, but it's generally a good practice to do relative imports with a dot: "from . Import exceptions as ResinException"